modules/core/pom.xml

```diff
@@ -83,6 +83,14 @@
       <artifactId>zookeeper</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.hdrhistogram</groupId>
```

Comment:
HdrHistogram is BSD and hdrhistogram-metrics-reservoir is Apache 2.0 so it sounds like it should not be an issue.